src/BroadcastMsg.chpl

```diff
@@ -28,6 +29,12 @@ module BroadcastMsg {
                                        getModuleName(),
                                        "TypeError");
     }
+
+    const objType = msgArgs.getValueOf("objType");
+    if objType != "pdarray" {
+      return broadcastStrings(cmd, msgArgs, st);
+    }
```

Comment:
u should the `ObjType` enum instead of doing string comparisons, something like:
```chapel
if objType.toUpper(): ObjType == ObjType.STRINGS {
    return broadcastStrings(cmd, msgArgs, st);
}
```

Reply:
Definitely agree here